data/json/recipes/weapon/mods.json

```diff
@@ -595,5 +595,64 @@
     "book_learn": [ [ "mag_archery", 1 ], [ "manual_archery", 1 ] ],
     "qualities": [ { "id": "CUT", "level": 1 } ],
     "components": [ [ [ "plastic_chunk", 1 ], [ "fur", 1 ] ], [ [ "superglue", 1 ], [ "bone_glue", 1 ] ] ]
+  },
+  {
+    "type": "recipe",
+    "result": "cord_crossbow_mech",
+    "category": "CC_WEAPON",
+    "subcategory": "CSC_WEAPON_MODS",
+    "skill_used": "fabrication",
+    "skills_required": [ [ "gun", 2 ] ],
+    "difficulty": 3,
+    "autolearn": true,
+    "time": 3000,
+    "qualities": [ { "id": "CUT", "level": 1 } ],
+    "components": [ [ [ "plastic_chunk", 1 ], [ "2x4", 1 ] ], [ [ "cordage", 1, "LIST" ] ] ]
+  },
+  {
+    "type": "recipe",
+    "result": "windlass_crossbow_mech",
+    "category": "CC_WEAPON",
+    "subcategory": "CSC_WEAPON_MODS",
+    "skill_used": "fabrication",
+    "skills_required": [ [ "gun", 3 ] ],
+    "difficulty": 5,
+    "autolearn": true,
+    "time": 30000,
+    "qualities": [
+      { "id": "CUT", "level": 1 },
+      { "id": "HAMMER", "level": 1 },
+      { "id": "SCREW", "level": 1 },
+      { "id": "WRENCH", "level": 1 },
+      { "id": "SAW_M", "level": 1 }
+    ],
+    "components": [ [ [ "rope_6", 2 ] ], [ [ "foot_crank", 1 ] ], [ [ "spring", 4 ] ], [ [ "steel_chunk", 4 ], [ "scrap", 12 ] ] ]
```

Comment:
Should make use of recently added rope requirements; either "rope_natural_short" or "rope_superior_short". I think the former would do, though, as it has nothing to do with the crossbow string and doesn't need to be as precise.

Same for recipe below.